onboard/cv/obstacle_detector.cpp

```diff
@@ -86,7 +107,8 @@ pair<int, float> get_final_col(vector<pair<int, float> > & sorted_sums, float mi
     // go straight if each direction has similar value
     return make_pair(-1, middle_sum);
 
-  } else {
+  } 
+  
```

Comment:
Why did you remove the `else` here? (Just curious, I have no issue with it at the moment but just wondering b/c this seems like it might be a logic change, as opposed to a refactoring change)

Reply:
I don't think it changed the logic, since the else was empty. I thought it just seemed cleaner without it.

Reply:
I don't think the else was empty? I looked here: https://github.com/umrover/mrover-workspace/blob/master/onboard/cv/obstacle_detector.cpp#L89 and it seems like lines 89-101 were all contained within the else statement.